crates/admin/src/rest_api/error.rs

```diff
@@ -23,6 +23,132 @@ use restate_types::invocation::ServiceType;
 use schemars::JsonSchema;
 use serde::Serialize;
 
+// --- Few helpers to define Admin API errors.
+
+/// Macro to generate an Admin API Error enum with the given variants.
+/// This implements both axum IntoResponse and okapi_operation ToResponses.
```

Comment:
Someone once wrote in a blog post that macros should be well documented. Maybe we should follow this lead?

Reply:
LOL touche'